ui/src/tests/expectExtenders.ts

```diff
@@ -0,0 +1,44 @@
+import { invariant } from '../util/invariant';
+
+export const toBeEnabled = (field: HTMLElement | null | Element) => {
+    invariant(field);
+
+    if (field.getAttribute('readonly')) {
+        return { pass: false, message: () => 'Field contains "readonly" attribute' };
+    }
+
+    const ariaDisabled = field.getAttribute('aria-disabled');
+
+    if (ariaDisabled === 'false') {
+        return { pass: true, message: () => 'Field is enabled' };
+    }
+
+    return {
```

Comment:
it does not have proper type checks so it may break if Jest changes the signature.
Can you extend jest in place with `jest.extend` and import this file from jest.setup.ts